README.md

```diff
@@ -19,7 +19,7 @@ Or [download a zip of the repository](https://github.com/scottaohara/accessible_
 The CSS for this component is included in `assets/css/`. The classes are added to the base markup when the script/page loads. The base CSS has only the most necessary styling to visually convey it as a modal dialog. You *will need to* modify the CSS to integrate the dialog into your project's visual aesthetic.  
 
 ## Standard Usage  
-Include the _a11y.modal.js_ file at the bottom of your document, or concatenated into your primary .js file, as part of your build process.
+Include the _aria.modal.min.js_ file at the bottom of your document, or concatenated into your primary .js file, as part of your build process.
```

Comment:
i actually want to remove the min file from this repo as people can just do that themselves.  

other than this, i'm good with the other updates.  thanks :)

Reply:
not a problem; will make a new PR.